.github/workflows/user_project_ci.yml

```diff
@@ -58,12 +57,17 @@ jobs:
         uses: docker/setup-buildx-action@v1
 
       - name: Export PDK ROOT
-        run: |
-          echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
-      
-      - name: Export PDKPATH
+        run: echo "PDK_ROOT=/home/runner/work/pdk" >> $GITHUB_ENV
+
+      - name: Export PDK PATH
         run: echo "PDKPATH=/home/runner/work/pdk/sky130A" >> $GITHUB_ENV
```

Comment:
is this line still needed ? what does PDKPATH get used by, and shouldn't it be PDK_ROOT?

Reply:
Used for precheck @a-omla can comment